src/VisualStudio/Core/Def/SymbolSearch/VisualStudioSymbolSearchService.LogService.cs

```diff
@@ -1,62 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
old service which got simplified and now just lives in the actual process we do the index updating in.